### PR TITLE
Update solanin.yaml (Blog 9 & Team Prop)

### PR DESCRIPTION
**Pull request by solanin**

### CODE REVIEW

File: people/2017/spring/solanin.yaml [hunk 1]
```diff
@@ -25,11 +25,14 @@ hw:
   blog05: http://hfoss-solanin.blogspot.com/2017/03/innovation.html
   blog06: http://hfoss-solanin.blogspot.com/2017/03/blog-post_12.html
   blog07: http://hfoss-solanin.blogspot.com/2017/03/nasa.html
+  blog08: http://hfoss-solanin.blogspot.com/2017/04/sexism.html
```
Comment (jflory7) [inline people/2017/spring/solanin.yaml]: Looks like this URL also has the wrong month? As-is, I wasn't able to find it, but changing `/04/` to `/03/` did the trick for me.

**Comment by solanin** [comment]
Resolved, deleted patch-1 branch and it's associated pull request. Since it had not been accepted change was put into master and is now part of this one (exclusively)

**Comment by solanin** [comment]
Now also contains blog 8, blog 10 and meetup 4

File: people/2017/spring/solanin.yaml (hunk 1 above)
Comment (solanin) [inline people/2017/spring/solanin.yaml]: done

**Pull request merged**